### DISCUSSION TITLE
Update mysql 5.7 for 8.0 in phalcon 4.1

### DISCUSSION THREAD

**Discussion by gustavomazzafc** [phalcon/cphalcon#16580] [Q&A]
Hi, i have a problem in update mysql version of my application, 8.0 version causes a 300% more loewst process in database compare of 5.7 version.

Any sugestions or cases for improve performance in this problem?

**Comment by Izopi4a** [comment]
from what i have seen accoss the internet mysql8 is just slower

**Reply by gustavomazzafc** [reply]
Yes, but it's much slower, it doesn't seem normal
